app/services/EmailMessageService.scala

```diff
@@ -50,7 +50,7 @@ class EmailMessageService @Inject()(emailMessageQueueRepository: EmailMessageQue
 
   def processWorkItem(acc: Seq[SecureCommsMessageModel],
                       workItem: WorkItem[SecureCommsMessageModel]): Future[Seq[SecureCommsMessageModel]] = {
-    try {
+
```

Comment:
Sorry man, can you put this try catch block back in place? After digging through the chain of functions it's possible that we throw an exception in the function `mapTemplateId` in `EmailService` (if the template ID is unrecognised).

I would argue that function is written badly and was going to suggest a small refactor so it cannot throw an exception, but it would have knock on effects to other functions and the type of information we log, so I think it's far out of the scope of this story. For now, if you could put the try catch back in place, and in your tests can you try to cover this scenario using a `when(x).thenThrow(new Exception("x"))` style mock? The existing mock in this test spec is a Future failed exception which will fall into the recover, but we just want a standard exception to hit the catch. Let me know if that does not make sense.